src/coreclr/jit/hwintrinsic.h

```diff
@@ -1133,6 +1138,48 @@ struct HWIntrinsicInfo
         }
     }
 
+#ifdef FEATURE_MASKED_HW_INTRINSICS
+    static bool HasAllMaskVariant(NamedIntrinsic id)
+    {
+        const HWIntrinsicFlag flags = lookupFlags(id);
+        return (flags & HW_Flag_HasAllMaskVariant) != 0;
+    }
+
+    static NamedIntrinsic GetMaskVariant(NamedIntrinsic id)
```

Comment:
i know lot of methods in this file don't have method summary, but we try to add them for newly added methods. So please add a line or 2 for them.